bokeh/models/annotations.py

```diff
@@ -338,6 +339,27 @@ class Label(Annotation):
     """)
 
 
+class Title(Label):
+    x = Override(default=0)
+
+    y = Override(default=0)
+
+    x_units = Override(default='screen')
+
+    y_units = Override(default='screen')
+
+    text_font_size = Override(default={'value' : '14pt'})
+
+    text_font_style = Override(default='bold')
+
+    def __init__(self, *args, **kwargs):
+        if len(args) == 1:
+            kwargs['text'] = [args[0]]
+        elif len(args) > 1:
+            raise ValueError("Too many positional arguments, expecting title text only")
+        super(Title, self).__init__(**kwargs)
+
+
```

Comment:
Given all the other properties suggestions, Title will not be able to subclass from Label.